include/rocksdb/options.h

```diff
@@ -746,6 +746,8 @@ struct DBOptions {
   // Default: 16MB/s
   uint64_t delayed_write_rate = 16 * 1024U * 1024U;
 
+  bool enable_pipeline_write = false;
```

Comment:
I guess in the next commit you will add comments to explain what this option means?